app/tests/evaluation_tests/test_views.py

```diff
@@ -1668,3 +1668,32 @@ def test_submission_create_sets_limits_correctly_with_predictions(client):
 
     assert submission.algorithm_requires_gpu_type == ""
     assert submission.algorithm_requires_memory_gb == 0
+
+
+@pytest.mark.django_db
+def test_phase_archive_info_permissions(client):
```

Comment:
We need to test cross-challenge permissions, that the correct phase is check for permissions for phases with the same slugs but different challenges.